lib/tasks/webpacker/install.rake

```diff
@@ -4,6 +4,7 @@ bin_path = ENV["BUNDLE_BIN"] || Rails.root.join("bin")
 namespace :webpacker do
   desc "Install Webpacker in this application"
   task install: [:check_node, :check_yarn] do |task|
+    @@webpacker_installing = true
```

Comment:
@dhh is this the right way to pipe the flag over that we're doing the installation?